bazel/envoy_internal.bzl

```diff
@@ -26,7 +26,7 @@ def envoy_copts(repository, test = False):
         "-Zc:__cplusplus",
         "-DWIN32",
         "-D_WIN32_WINNT=0x0A00",  # _WIN32_WINNT_WIN10
-        "-DNTDDI_VERSION=0x0A000000",  # NTDDI_WIN10
+        "-DNTDDI_VERSION=0x0A000005",  # NTDDI_WIN10_RS4
```

Comment:
Should we document this somewhere? That you will need a pretty up to date Windows 10/Server 2019 version?

Reply:
RS4 was released at Q1 of 2018. Compilation will fail before that anyway because the `af_unix` header does not exist.  Don't we do that already?

Reply:
ok sweet, that sounds good to me then if RS4 is old enough

Reply:
Added also a comment in the readme file.